modules/installing-ocp-agent-ibm-z.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent-based-installer/installing-with-agent-based-installer.adoc
+// * installing/installing_with_agent_based_installer/prepare-pxe-infra-agent.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-ocp-agent-ibm-z_{context}"]
+= Manually adding {ibm-z-title} agents
+
+After creating the PXE assets you can add {ibm-z-name} agents. Depending on your {ibm-z-name} environment you can choose from the following:
+
+* Adding {ibm-z-name} agents with z/VM
+* Adding {ibm-z-name} agents with {op-system-base} KVM
+
+.Procedure
+
+** Adding {ibm-z-name} agents with z/VM
+
+. Create a parameter file for the z/VM guest.
++
+.Example parameter file
++
+[source,terminal]
+----
+rd.neednet=1 \
+console=ttysclp0 \
+coreos.live.rootfs_url=<rootfs_url> \ <1>
+ip=172.18.78.2::172.18.78.1:255.255.255.0:::none nameserver=172.18.78.1 \ <2>
+zfcp.allow_lun_scan=0 \ <3>
+rd.znet=qeth,0.0.bdd0,0.0.bdd1,0.0.bdd2,layer2=1 \
+rd.dasd=0.0.4411 \ <4>
+rd.zfcp=0.0.8001,0x50050763040051e3,0x4000406300000000 \ <5>
+random.trust_cpu=on rd.luks.options=discard \
+ignition.firstboot ignition.platform.id=metal \
+console=tty1 console=ttyS1,115200n8 \
+coreos.inst.persistent-kargs="console=tty1 console=ttyS1,115200n8"
+----
+<1> For the `coreos.live.rootfs_url=` artifact, specify the matching `rootfs` artifact for the `kernel`and `initramfs` you are booting. Only HTTP and HTTPS protocols are supported.
+<2> For the `ip=` parameter, assign the IP address automatically using DHCP or manually as described in "Installing a cluster with z/VM on {ibm-z-name} and {ibm-linuxone-name}".
+<3> The default is '1'. Omit this entry when using an OSA network adapter.
+<4> For installations on DASD-type disks, use `rd.dasd=` to specify the DASD where {op-system} is to be installed. Omit this entry for FCP-type disks.
+<5> For installations on FCP-type disks, use `rd.zfcp=<adapter>,<wwpn>,<lun>` to specify the FCP disk where {op-system} is to be installed. Omit this entry for DASD-type disks.
++
+Leave all other parameters unchanged.
+
+. Punch the `kernel.img`,`generic.parm`, and `initrd.img` files to the virtual reader of the z/VM guest virtual machine.
++
+See link:https://www.ibm.com/docs/en/zvm/latest?topic=commands-punch[PUNCH] in IBM Documentation.
++
+[TIP]
+====
+You can use the CP PUNCH command or, if you use Linux, the **vmur** command to transfer files between two z/VM guest virtual machines.
+====
++
+. Log in to CMS on the bootstrap machine.
```

Comment:
What is meant by CMS? Is it a guest vm console to zvm?

Reply:
Yes see CMS Guide [here](chrome-extension://efaidnbmnnnibpcajpcglclefindmkaj/https://www-40.ibm.com/servers/resourcelink/svc0302a.nsf/pages/zVMV7R2sc246266/$file/dmsb3_v7r2.pdf).